examples/README.md

```diff
@@ -1,229 +1,60 @@
-# Supabase `storage-php`
+# Supabase `storage-php` examples 
 
-PHP Client library to interact with Supabase Storage.
+These examples will provide a way to interact with the `storage-php` library.
 
-## Quick Start Sample Guide
+```
+.
+├── handling-buckets
+│   ├── create-bucket.php
+│   ├── delete-bucket.php
+│   ├── empty-bucket.php
+│   ├── get-bucket.php
+│   ├── list-buckets.php
+│   └── update-bucket.php
+└── handling-files
+    ├── copy-file.php
+    ├── create-signed-url-file.php
+    ├── download-file.php
+    ├── get-public-url-file.php
+    ├── list-files.php
+    ├── move-file.php
+    ├── remove-file.php
+    ├── update-file.php
+    └── upload-file.php
+```
 
-### Installing the module
+## Setup
 
-```bash
-composer require supabase/storage-php
-```
+### **Downloading the package**
+Download the package supabase/storage-php. On your terminal run `composer require supabase/storage-php` 
 
-### Connecting to the storage backend
+### **API Keys**
+To get your API keys, please sign into your Supabase account. 
+Once signed on your dashboard, go to, Project >> Project Settings >> API Settings >> Project API keys
+ #### *You can use the keys to use Supabase client libraries.*
 
```

Comment:
Let's add a bit about copying the `env.example` to `env.local` an example here: https://github.com/supabase/supabase-js/tree/master/examples/next-ts